shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-opengauss/src/main/antlr4/imports/opengauss/BaseRule.g4

```diff
@@ -1276,6 +1276,18 @@ aliasClause
     | colId
     ;
 
+directoryname
```

Comment:
Do you think `directoryName` is better?

Reply:
agreed, we need to maintian consistency. made the changes